server/models/SentenceAndOffenceViewModel.ts

```diff
@@ -56,20 +55,13 @@ export default class SentenceAndOffenceViewModel {
   }
 
   public rowIsSdsPlus(sentence: AnalyzedSentenceAndOffences, offence: PrisonApiOffenderOffence): boolean {
-    const input =
+    const oldUserInputForSDSPlus =
       this.userInputs &&
-      this.userInputs.sentenceCalculationUserInputs.find((it: CalculationSentenceUserInput) => {
+      this.userInputs.sentenceCalculationUserInputs?.find((it: CalculationSentenceUserInput) => {
         return it.offenceCode === offence.offenceCode && it.sentenceSequence === sentence.sentenceSequence
       })
-    return input && input.userChoice
-  }
-
-  public lastUserQuestion(): CalculationQuestionTypes {
-    if (this.userInputs) {
-      const userInputTypes = CalculationQuestionTypes.getOrderedQuestionTypesFromInputs(this.userInputs)
-      return userInputTypes[userInputTypes.length - 1]
-    }
-    return null
+    const isUserIdentifiedSDSPlus = oldUserInputForSDSPlus && oldUserInputForSDSPlus.userChoice
+    return isUserIdentifiedSDSPlus || offence.indicators?.includes('PCSC/SDS+')
```

Comment:
Can this use isPcscSdsPlus instead of maintaining this string in front & back ends?